rules/linux/privilege_escalation_awk_binary_shell.toml

```diff
@@ -0,0 +1,76 @@
+[metadata]
+creation_date = "2022/02/24"
+maturity = "production"
+updated_date = "2022/02/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Linux binary mawk or nawk abuse to breakout out of restricted shells or environments by spawning an
+interactive system shell.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Linux Restricted Shell Breakout via nawk /mawk Shell Evasion"
+references = ["https://gtfobins.github.io/gtfobins/nawk/",
+              "https://gtfobins.github.io/gtfobins/mawk/"]
+risk_score = 47
+rule_id = "10754992-28c7-4472-be5b-f3770fd04f2d"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Privilege Escalation","GTFOBins"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+sequence by host.id, process.pid with maxspan=1m
+[process where process.name : ("nawk", "mawk", "awk", "gawk") and process.args: "BEGIN {system(*)}"]
+[process where process.parent.name : ("nawk", "mawk", "awk", "gawk") and process.name : "sh"]
+'''
+
+
+[[rule.threat]]
```

Comment:
only technique I see here is T1059.004, maybe https://attack.mitre.org/techniques/T1127/ too

Reply:
**Corrected**